### PR TITLE
#85 unit test for type serializer

### LINKED ISSUE #85: Tests for OnMoon\OpenApiServerBundle\Types\TypeSerializer
Cover OnMoon\OpenApiServerBundle\Types\TypeSerializer with unit tests

### PR DESCRIPTION
**Pull request by egorzot**
Closes #85

### CODE REVIEW

File: test/unit/Types/TypeSerializerTest.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+namespace OnMoon\OpenApiServerBundle\Test\Types;
+
+use DateTime;
+use OnMoon\OpenApiServerBundle\Types\TypeSerializer;
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\TestCase;
+use Safe\Exceptions\DatetimeException;
+use Throwable;
+
```
Comment (sspat) [inline test/unit/Types/TypeSerializerTest.php:13]: Please add @covers annotation

File: test/unit/Types/TypeSerializerTest.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    public function testSerializeDateTimeReturnsSerializedDateTime(): void
+    {
+        $dateString             = '25-12-2020 15:19:21';
+        $expectedSerializedDate = '2020-12-25T15:19:21+00:00';
+        $date                   = new \Safe\DateTime($dateString);
+        $serializedDate         = TypeSerializer::serializeDateTime($date);
+        Assert::assertSame($expectedSerializedDate, $serializedDate);
+    }
+
+    public function testDeserializeByteReturnsDesirializedString(): void
```
Comment (sspat) [inline test/unit/Types/TypeSerializerTest.php:64]: There is a typo in "Sirialized" in a couple of places

File: test/unit/Types/TypeSerializerTest.php (hunk 1 above)
Comment (egorzot) [reply test/unit/Types/TypeSerializerTest.php:13]: fixed

File: test/unit/Types/TypeSerializerTest.php (hunk 2 above)
Comment (egorzot) [reply test/unit/Types/TypeSerializerTest.php]: fixed

**Pull request merged**